framework/doc/content/source/meshgenerators/RenameBoundaryGenerator.md

```diff
@@ -2,55 +2,58 @@
 
 !syntax description /Mesh/RenameBoundaryGenerator
 
-## Overview
+## Renaming or Setting Boundary Names
 
-`RenameBoundaryGenerator` is usually used to provide meaningful names to boundaries.
-For instance
+When using the `RenameBoundaryGenerator` to change boundary names, the result is independent of ordering.
 
-```text
-old_boundary_id = '1 2 3'
-new_boundary_name = 'right top left'
-```
-
-Then the MOOSE input file can employ `boundary = right` rather than `boundary = 1`.  `RenameBoundaryGenerator` may also
-be used to provide more meaningful names to boundaries that are already named.  For instance
+The following will change the name for the boundary `meaningless` to `interior` and the name for boundary `5` to `exterior`:
```

Comment:
I think meaningless should be monospace, since this is really just a string. I like to use regular quotes for this and save the monospace for actual code snippets and class names (which are code snippets).